Makefile

```diff
@@ -90,6 +90,10 @@ sandbox-release: neard-sandbox-release
 neard-sandbox-release:
 	cargo build -p neard --features sandbox --release
 
+statelessnet-release: NEAR_RELEASE_BUILD=release
+statelessnet-release:
```

Comment:
I would perhaps have added a variable to pick between a regular, sandbox or statelessnet versions, but given that other combinations are already setting a poor prior art, this is fine and I guess any changes would be a refactor instead.

(primarily because the matrix of possible builds is a `{ nightly, regular } x { debug, release } x { regular, statelessnet, sandbox, perf, ... }` etc. and adding individual rules for each of these is going to blow up the complexity of the makefile significantly.